src/components/overview/MobileClientCardView.js

```diff
@@ -11,7 +13,8 @@ class MobileClientCardView extends Component {
     this.state = { filter: '', currentValue: '' };
 
     this.emptyStateMessage = {
-      noAppsCreated: 'You have no mobile apps right now. Create one to get started.',
+      noAppsCreatedTitle: "You don't have any Mobile Apps.",
```

Comment:
Our [eslint configuration](https://github.com/aerogear/mobile-developer-console/blob/c151bb4532b63358224f41bee47a4ef78aa55850/.eslintrc.json) is set to prefer single quotes for JS files. I'm not sure why, but eslint is not actually picking up that specific line for some reason.

See below, I changed the next property in the object which gives an error, but not in `noAppsCreatedTitle`.

![Screenshot from 2019-08-01 10-08-34](https://user-images.githubusercontent.com/11743717/62280746-6eba7500-b444-11e9-8c77-ff08691fce37.png)

Reply:
I'll update these to single quotes and fix the button.